src/test/c_api_stress_tests.cpp

```diff
@@ -275,7 +357,7 @@ TEST_F(StressCapiConfigChanges, AddModelDuringGetModelStatusLoad) {
 
 class StressPipelineCustomNodesWithPreallocatedBuffersCapiConfigChanges : public StressCapiConfigChanges {
 public:
-    void checkInferResponse(OVMS_InferenceResponse* response) override {
+    void checkInferResponse(OVMS_InferenceResponse* response, std::string& expectedOutputName) override {
```

Comment:
Not used?

Reply:
Yes, but it is needed to override and proper virtual call.